src/RequiredNetFX46FacadeAssemblies.targets

```diff
@@ -1,17 +1,23 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   
+  <PropertyGroup>
```

Comment:
These changes should be reverted and instead just exclude this stuff for the mono/desktop configuration.